README.md

```diff
@@ -13,36 +13,76 @@ for use within the [HCP Minimal Preprocessing Pipelines][HCP Pipelines].
 
 ## Installation
 
-### Prerequisites
-
+### Install for all users
 You can install the necessary prerequisites in most Ubuntu or Debian-based distributions with this command:
-```
-sudo apt-get install python3-numpy python3-pip
+```bash
+sudo apt install python3-numpy python3-pip
```

Comment:
Is it a good idea to have python3-numpy in here, when numpy is in the requirements.txt?

Reply:
I just checked, and it doesn't seem to really matter. 
If `numpy` and `scipy` are already installed (e.g. because `apt install python3-numpy` was run), then the `-r requirements.txt` step just says e.g.
```
Requirement already satisfied: numpy in /usr/lib/python3/dist-packages
```
and if they are not then it installs the latest versions.